frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -101,20 +108,33 @@ export default function CellGuideCard({
   const { organismsList, organsMap } =
     useOrganAndOrganismFilterListForCelltype(cellTypeId);
 
-  const uniqueOrgans = Array.from(organsMap.keys());
+  const sdsOrganismsList = organismsList.map((organism) => ({
```

Comment:
let's memoize this, so we don't create a new array every time, which is a performance issue, since `[] === [] // false`